src/tests/system/tests/test_ipa.py

```diff
@@ -0,0 +1,91 @@
+"""
+SSSD IPA tests.
+
+:requirement: ipa
+"""
+
+from __future__ import annotations
+
+import time
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ipa import IPA
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa_add_support_to_verify_authentication_indicators(client: Client, ipa: IPA):
+    """
+
+    Kerberos client may have different means to prove possession of a client principal
```

Comment:
No, this is bad form. I don't want to have to read something verbatim when I click on a link. The description should try to summarize that page into something that helps other people on the team have a better idea of what the test does. 

"authentication indicators is a marker? that tracks the pre-authenticated method that was used" 

I think @aplopez worked on this, so let's get his opinion?

but create as :description: in the docstring  after :title:

Also, you don't need to provide the link.